server/events/event_parser.go

```diff
@@ -112,6 +112,10 @@ func (c CommentCommand) IsForSpecificProject() bool {
 	return c.RepoRelDir != "" || c.Workspace != "" || c.ProjectName != ""
```

Comment:
Please re-word the PR summary and title to reflect that this is no longer using the `-f` flag and instead will add wildcard filtering using the existing `-d` flag